src/vm/wren_meta.h

```diff
@@ -3,6 +3,7 @@
 
 #include "wren.h"
 #include "wren_common.h"
+#include "wren_vm.h"
```

Comment:
Why add this include in `wren_meta.h` rather than `wren_meta.c`? It looks like it should suffice to have it in the source file.